src/scss/components/_footer.scss

```diff
@@ -35,7 +35,12 @@
     @include media(medium) {
       text-align: inherit;
     }
+
     color: get-color((color: air, opacity: .5));
+    font: {
+      size: 11px;
+      weight: 400;
```

Comment:
@eduardojmatos Can we use the get-weight helper here?

Reply:
@iondrimba hmm I think yes. This technical debt should have been paid when the peso was created. But I'll ship it on this PR